### PR TITLE
Add "Disable Background Parallax On Mobile" Global Setting

### PR DESCRIPTION
**Pull request by AlexGStapleton**
Resolve https://github.com/siteorigin/siteorigin-panels/issues/158

Requires premium PR: https://github.com/siteorigin/siteorigin-premium/pull/403
The premium is required because it overrides the PB parallax with its own (modified) verison.

### CODE REVIEW

File: inc/settings.php [hunk 1]
```diff
@@ -284,6 +284,12 @@ function settings_fields( $fields ) {
 			'description' => __( 'How many pixels of scrolling result in a single pixel of parallax motion. 0 means automatic. Lower values give more noticeable effect.', 'siteorigin-panels' ),
 		);
 
+		$fields['general']['fields']['parallax-mobile'] = array(
+			'type'        => 'checkbox',
+			'label'       => __( 'Disable Background Parallax On Mobile', 'siteorigin-panels' ),
+			'description' => __( 'Disable row/widget background parallax when browser is smaller than the mobile width.', 'siteorigin-panels' ),
```
Comment (Misplon) [inline inc/settings.php:290]: Let's perhaps change this to: `Disable row/widget background parallax when the browser is smaller than the mobile width.`

It's quite long already so adding `the` won't hurt I don't think.

File: inc/settings.php [hunk 2]
```diff
@@ -284,6 +284,12 @@ function settings_fields( $fields ) {
 			'description' => __( 'How many pixels of scrolling result in a single pixel of parallax motion. 0 means automatic. Lower values give more noticeable effect.', 'siteorigin-panels' ),
 		);
 
+		$fields['general']['fields']['parallax-mobile'] = array(
+			'type'        => 'checkbox',
+			'label'       => __( 'Disable Background Parallax On Mobile', 'siteorigin-panels' ),
```
Comment (Misplon) [inline inc/settings.php:289]: The setting above is "Limit Parallax Motion". What do you think about changing this label to "Disable Parallax On Mobile" ?

File: inc/styles.php [hunk 3]
```diff
@@ -54,14 +54,18 @@ static function register_scripts() {
...
 		wp_register_script(
 			'siteorigin-parallax',
 			siteorigin_panels_url( 'js/siteorigin-parallax' . SITEORIGIN_PANELS_JS_SUFFIX . '.js' ),
 			array( 'jquery' ),
 			SITEORIGIN_PANELS_VERSION
 		);
-		wp_localize_script( 'siteorigin-panels-front-styles', 'panelsStyles', array(
-			'fullContainer' => apply_filters( 'siteorigin_panels_full_width_container', siteorigin_panels_setting( 'full-width-container' ) )
+		wp_localize_script( 'siteorigin-parallax', 'parallaxStyles', array(
+			'parallax-mobile' => ! empty( siteorigin_panels_setting( 'parallax-mobile' ) ) ? siteorigin_panels_setting( 'parallax-mobile' ) : false,
```
Comment (gregpriday) [inline inc/styles.php:67]: Can we use `?:` with the version of PHP we're supporting yet?

Comment (AlexGStapleton) [reply inc/styles.php]: Yes. Just submitted a commit using that instead..

**Pull request merged**